### ISSUE TITLE
Unknown command: trigger-job 

### ISSUE THREAD

**Issue by annaken** [troykinsella/concourse-fly-resource#15]
I'm trying to trigger one job from a different pipeline like 
```
- put: trigger-acceptance-tests
  params:
    options: trigger-job -j acceptance-tests/run-tests
```
But when I run that in Concourse I get:

> error: Unknown command `trigger-job -j acceptance-tests/run-tests'. Please specify one command of: abort-build, builds, check-resource, check-resource-type, checklist, clear-task-cache, containers, curl, delete-target, destroy-pipeline, destroy-team, edit-target, execute, expose-pipeline, format-pipeline, get-pipeline, get-team, help, hide-pipeline, hijack, jobs, land-worker, login, logout, order-pipelines, pause-job, pause-pipeline, pipelines, prune-worker, rename-pipeline, rename-team, resource-versions, resources, set-pipeline, set-team, status, sync, targets, teams, trigger-job, unpause-job, unpause-pipeline, userinfo, validate-pipeline, volumes, watch or workers

If I try with just `options: trigger-job` I get `error: the required flag -j, --job was not specified`, so I'm completely baffled as to why this is failing. 

I am definitely logged in ok, if I do `options: sync` then that works fine.

**Comment by troykinsella** [comment]
@annaken Seems I was over-aggressive with quoting! This should be fixed on the `latest`, `develop`, and `2.0.1` tags.  Please re-test.

**Comment by annaken** [comment]
Yep that works! Thanks :)
